src/app/examples/si-chat-messages/si-user-message.ts

```diff
@@ -20,8 +22,28 @@ import { LOG_EVENT } from '@siemens/live-preview';
 export class SampleComponent {
   logEvent = inject(LOG_EVENT);
   private sanitizer = inject(DomSanitizer);
+  private translate = injectSiTranslateService();
 
-  protected markdownRenderer = getMarkdownRenderer(this.sanitizer);
+  protected markdownRenderer = getMarkdownRenderer(this.sanitizer, {
+    copyCodeButton: 'SI_MARKDOWN_RENDERER.COPY',
+    downloadTableButton: 'SI_MARKDOWN_RENDERER.DOWNLOAD',
```

Comment:
Don't include this in the example as the properties are defaulted.

```suggestion
```

Reply:
And here